src/lib/y2network/widgets/address_tab.rb

```diff
@@ -0,0 +1,102 @@
+require "yast"
+require "cwm/tabs"
+
+# used widgets
+require "y2network/widgets/additional_addresses"
+require "y2network/widgets/boot_protocol"
+require "y2network/widgets/ip_address"
+require "y2network/widgets/netmask"
+require "y2network/widgets/remote_ip"
+require "y2network/widgets/tunnel"
+require "y2network/widgets/vlan_id"
+require "y2network/widgets/vlan_interface"
+
+module Y2Network
+  module Widgets
+    class AddressTab < CWM::Tab
+      def initialize(settings)
+        textdomain "network"
+
+        @settings = settings
+      end
+
+      def label
+        _("&Address")
+      end
+
+      def contents
+        type = @settings.type
+
+        drvtype = driver_type(type)
+        is_ptp = drvtype == "ctc" || drvtype == "iucv"
```

Comment:
Add a #FIXME or comment to check if we can already remove this interfaces.

See:
https://www.suse.com/documentation/sles-15/book_sle_deployment/data/sec_zseries_requirements.html
https://www.suse.com/documentation/sles-15/book_sle_admin/data/sec_network_yast.html#sec_network_yast_s390_ctc